test/unit/views/IleQueryDucrResponseViewSpec.scala

```diff
@@ -18,33 +18,40 @@ package views
 
 import java.time.ZonedDateTime
 
+import base.Injector
 import models.notifications.EntryStatus
-import models.notifications.queries.{DucrInfo, MovementInfo}
+import models.notifications.queries.{DucrInfo, MovementInfo, MucrInfo}
+import models.viewmodels.decoder.ROECode.UnknownRoe
+import models.viewmodels.decoder.{ICSCode, ROECode, SOECode}
 import play.api.mvc.{AnyContent, Request}
 import play.api.test.FakeRequest
+import play.twirl.api.Html
 import views.html.ile_query_ducr_response
 
-class IleQueryDucrResponseViewSpec extends ViewSpec {
+class IleQueryDucrResponseViewSpec extends ViewSpec with Injector {
```

Comment:
There is a lot of tests in here. Might be worth creating tests for the components (page parts) and then only check if this view calls them with correct parameters.